pkg/cloud/volume_limits.go

```diff
@@ -67,6 +67,7 @@ var nonNitroInstanceFamilies = map[string]struct{}{
 	"g3":  {},
 	"d2":  {},
 	"h1":  {},
+	"f1":  {},
```

Comment:
Are we sure f1 is not nitro instance family? 

I see it in the accelerated computing current gen instance list: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-types.html#ec2-nitro-instances

But perhaps I'm looking in wrong place

Reply:
if you run `❯ aws ec2 describe-instance-types --instance-types  f1.2xlarge` you will see that `            "Hypervisor": "xen",` and not nitro. As such I manually confirmed this by attaching 40 volumes to the  f1.2xlarge instance type

Reply:
I can confirm F1 definitely is non-nitro

Reply:
Thanks for the deep dive!